optuna/storages/_rdb/storage.py

```diff
@@ -671,7 +671,7 @@ def _update_trial(
                 .all()
             )
             param_keys = set(param.param_name for param in trial_param)
-            trial_model.params.extend(
+            trial_model._params.extend(
```

Comment:
This `trial_model` is not an instance of `FrozenTrial` but that of `TrialModel`. It is good to revert this change.

Reply:
Indeed this part caused errors... Thanks a lot!